client/src/components/FeedbackForm/FeedbackForm.js

```diff
@@ -0,0 +1,110 @@
+import React, { useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './FeedbackForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  subject: Yup.string().required('Required'),
+  name: Yup.string().required('Required'),
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Required'),
+  problem_description: Yup.string().required('Required')
+});
+
+const FeedbackForm = ({ authenticatedUser }) => {
+  const dispatch = useDispatch();
+  const creating = useSelector(state => state.ticketCreate.creating);
+  const creatingError = useSelector(state => state.ticketCreate.creatingError);
+  const creatingErrorMessage = useSelector(
+    state => state.ticketCreate.creatingErrorMessage
+  );
+
+  const defaultValues = useMemo(
+    () => ({
+      subject: 'Feedback for the Frontera Portal',
+      problem_description: '',
+      first_name: authenticatedUser ? authenticatedUser.first_name : '',
+      last_name: authenticatedUser ? authenticatedUser.last_name : '',
```

Comment:
Are these necessary?

Reply:
Not necessary, was part of outdated code, sorry.